celery_singleton/singleton.py

```diff
@@ -54,6 +57,10 @@ def get_existing_task_id(self, lock):
         return self.singleton_backend.get(lock)
 
     def generate_lock(self, task_name, task_args=None, task_kwargs=None):
+        unique_on = self.unique_on
+        return self._generate_lock_filtered_by_unique_on(unique_on, task_name, task_args, task_kwargs)
+    
+    def _generate_lock_filtered_by_unique_on(self, unique_on, task_name, task_args=None, task_kwargs=None):
```

Comment:
Why wrap generate_lock like this? I don't see the point in moving this method without any changes. Am I missing something.

Reply:
Sorry for that, this is not used. I am removing it !